frontend/src/App.tsx

```diff
@@ -28,7 +28,8 @@ const App = () => {
             <AuthContainer></AuthContainer>
           </Box>
           <Box smallIcon={"B"}>
-            <p>Number of Bits: {gameData.numBits}</p>
+            <p> Current Number of Bits: {gameData.numBits}</p>
+            <p> Total Number of Bits: {gameData.totalNumBits}</p>
```

Comment:
Idea: maybe we should call this "Lifetime bits" or something of that nature? Or maybe "total bits collected" or something? Just want to clarify the difference.